vagrant/release/devstack/Vagrantfile

```diff
@@ -28,6 +28,7 @@ MOUNT_DIRS = {
   :edx_platform => {:repo => "edx-platform", :local => "/edx/app/edxapp/edx-platform", :owner => "edxapp"},
   :themes => {:repo => "themes", :local => "/edx/app/edxapp/themes", :owner => "edxapp"},
   :comprehensive_themes => {:repo => "comprehensive_themes", :local => "/edx/app/edxapp/comprehensive_themes", :owner => "edxapp"},
+  :xblocks => {:repo => "xblocks", :local => "/edx/app/edxapp/xblocks", :owner => "edxapp"},
```

Comment:
@dcadams This should handle the mounting. 

You do _not_ need to rebuild the box to test this.
You should be able to:
- update/redownload the `Vagrantfile` locally
- `vagrant halt; vagrant up`